src/myr/Myr.js

```diff
@@ -421,7 +427,25 @@ class Myr {
         return this.cursor.color;
     }
 
-    getRandomColor = (colors = null) => {
+    setTexture = (texture = "blank") => {
+        let holdKey;
+
+        TexturePack.forEach(function(item) {
+            Object.keys(item).forEach(function(key) {
+                if(key === texture){
+                    holdKey = item[key];
+                }
+            });
+        });
```

Comment:
You should change your data type in TexturePack. This logic is way too complex for what you're doing. Your data type is also not following good practices. There are 2 options on how to do so:
```js
TexturePack={
"bark":"url.com/bark.jpg",  
"brick":"url.com/brick.jpg"  
}
```

or

```js
TexturePack=[  
{title:"bark",url:"url.com/bark.jpg"},  
{title:"brick",url:"url.com/brick.jpg"}  
]
```